ccp/apis/server/ccp_server/env_config.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+
+OPENSHIFT_SERVER = os.environ.get("OPENSHIFT_SERVER") or \
+                   "https://localhost:8443"
```

Comment:
no defaults for this value please, this makes things fuzzy.
eg: someone forgets to provide this var, in that case default value will be picked up (where OpenShift is not running), leading to hard debugging.